pallets/xcmp-queue/src/lib.rs

```diff
@@ -374,6 +376,16 @@ pub mod pallet {
 	/// Whether or not the XCMP queue is suspended from executing incoming XCMs or not.
 	#[pallet::storage]
 	pub(super) type QueueSuspended<T: Config> = StorageValue<_, bool, ValueQuery>;
+
+	frame_support::parameter_types! {
+		pub InitialFactor: FixedU128 = FixedU128::from_u32(1);
+	}
+
+	/// The number to multiply the base delivery fee by.
+	#[pallet::storage]
+	#[pallet::getter(fn delivery_fee_factor_xcmp)]
+	pub(crate) type DeliveryFeeFactorXCMP<T: Config> =
```

Comment:
```suggestion
	pub(crate) type DeliveryFeeFactorXcmp<T: Config> =
```

Reply:
Since you're in the XCMP pallet in this file, it doesn't require specifying which pallet that this delivery fee factor belongs to.